connectors/src/lib/error.ts

```diff
@@ -79,3 +79,11 @@ export class InvalidRowsRequestError extends Error {
     this.name = "InvalidRowsRequestError";
   }
 }
+
+// Error for body too large
+export class BodyTooLargeError extends Error {
```

Comment:
Could be worh having a single GoogleTablesError (or better named) with a type, such as done elsewhere in the codebase e.g. front/components/assistant/conversation/ConversationError.tsx